sys/key_rev_latest_value.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
+
+var zlib = require('zlib');
+var P = require('bluebird');
+var uuid = require('cassandra-uuid').TimeUuid;
+var preq = require('preq');
+
+var HyperSwitch = require('hyperswitch');
+var URI = HyperSwitch.URI;
+
+var spec = HyperSwitch.utils.loadSpec(__dirname + '/key_rev_value.yaml');
+
+function ArchivalBucket(options) {
+}
+
+ArchivalBucket.prototype._latestName = function(bucket) {
+    return bucket + '.latest';
+};
+
+ArchivalBucket.prototype._archiveName = function(bucket) {
+    return bucket;
+};
+
+ArchivalBucket.prototype.createBucket = function(hyper, req) {
+    var self = this;
+    var rp = req.params;
+    var latestConfig = Object.assign({}, req.body, {
+        revisionRetentionPolicy: { type: 'latest_hash' }
+    });
+    if (latestConfig.valueType !== 'json') {
+        latestConfig.options = latestConfig.options || {};
+        latestConfig.options.compression = [];
+    }
+    return P.join(
+        hyper.put({
+            uri: new URI([rp.domain, 'sys', 'key_rev_value', self._latestName(rp.bucket)]),
```

Comment:
The table backing this should not have any compression enabled (or at most something really fast like lzo).